### PR TITLE
Add SbtSlamdata plugin

### PR DESCRIPTION
**Pull request by drostron**

### CODE REVIEW

File: project/Publish.scala [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      else
+        Some("releases"  at nexus + "service/local/staging/deploy/maven2")
+    },
+    publishMavenStyle := true,
+    publishArtifact in Test := false,
+    pomIncludeRepository := { _ => false },
+    releasePublishArtifactsAction := PgpKeys.publishSigned.value,
+    releaseCrossBuild := true,
+    autoAPIMappings := true,
+    scmInfo := Some(
```
Comment (wemrysi) [inline project/Publish.scala]: Looks like this snuck by, should be removed.

Comment (drostron) [inline project/Publish.scala]: Trickster spot appreciated — ✄.

File: src/main/scala/slamdata/Publish.scala [hunk 2]
```diff
@@ -0,0 +1 @@
+../../../../project/Publish.scala
```
Comment (sellout) [inline src/main/scala/slamdata/Publish.scala]: What does this mean? Is it just GitHub’s way of showing me a symlink or something?

Comment (drostron) [inline src/main/scala/slamdata/Publish.scala]: Correct, it's a symlink.

**Comment by drostron** [comment]
Agreed on encrypted things — at this point we're just continuing with the approach we have in quasar and now we'll have a consolidated location if we come up with an alternative approach. Also, if we felt motivated enough, moving forward whomever submits something encrypted could pair on that so an additional set of eyes have verified the contents.

**Pull request merged**